backend/fms_core/services/library.py

```diff
@@ -255,24 +254,17 @@ def convert_library_concentration_from_ngbyul_to_nm(source_sample, concentration
     if source_sample is None:
         errors.append(f'Missing sample.')
 
-    final_concentration = 0
+    sum_adjusted_concentrations = 0
     for derived_sample in source_sample.derived_samples.all():
         # Compute the size of each library and its volume ratio
         library = derived_sample.library
         volume_ratio = DerivedBySample.objects.get(derived_sample=derived_sample, sample=source_sample).volume_ratio
         if library.library_size and library.strandedness:
             # Convert the concentration
-            partial_concentration = convert_concentration_from_ngbyul_to_nm(concentration=concentration_ngbyul,
-                                                                            molecular_weight=library.molecular_weight_approx,
-                                                                            molecule_count=library.library_size,
-                                                                            volume_ratio=volume_ratio)
-            if partial_concentration is None:
-                errors.append(f'Failed to convert the concentration of this library {source_sample.name}.')
-                return None, errors, warnings
-            else:
-                # Adjust the concentration according to its volume ratio
-                final_concentration += partial_concentration
+            adjusted_concentration = (library.library_size * library.molecular_weight_approx * volume_ratio)
```

Comment:
misleading name. This is not a concentration. it's some kind of partial adjusment factor. Not really sure of a good name for this but i would try not to use concentration since it may be confusing with already 2 other concentration variables.